plugins/ipc-rules/ipc-utility-methods.hpp

```diff
@@ -281,6 +283,16 @@ class ipc_rules_utility_methods_t
             return wf::ipc::json_error("Options must be an object!");
         }
 
+        const char *config_file     = std::getenv("WAYFIRE_CONFIG_FILE");
+        std::string config_file_str = config_file ? config_file : "wayfire.ini";
+
+        if (warning_count <= MAX_WARNINGS)
+        {
+            LOGW("Config options set via IPC will not update from ", config_file_str,
+                " until Wayfire restart.");
+            warning_count++;
```

Comment:
I would say we should ++warning_count every time, and when we hit MAX_WARNINGS, print a warning that we will suppress future warnings. Just so that users don't wonder why there are no subsequent warnings xd